compiler/rustc_metadata/src/creader.rs

```diff
@@ -308,7 +308,7 @@ impl<'a> CrateLoader<'a> {
             }
 
             // Alright, so we've gotten this far which means that `data` has the
-            // right name, we don't have a hash, and we don't have a --extern
+            // right name, we don't have a hash, and we don't have an --extern
```

Comment:
like in miri, I read this as "dash dash extern", so the "a" is correct, but I see how this change also can make sense

Reply:
Seems reasonable, I’ll remove the change.

Reply:
If we want to avoid commits touching the file, I can rebase the PR after review.